<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-NTDIJGK" ignored="0" name="Regression suite 1" tests="30" failures="4" timestamp="2020-06-18T19:14:25 ICT" time="81.742" errors="0">
  <testcase name="ACC05" time="0.788" classname="poly.test.Account"/>
  <testcase name="ACC06" time="0.797" classname="poly.test.Account"/>
  <testcase name="ACC07" time="2.185" classname="poly.test.Account"/>
  <testcase name="ACC08" time="1.652" classname="poly.test.Account"/>
  <testcase name="ACC09" time="0.779" classname="poly.test.Account"/>
  <testcase name="ACC04" time="0.715" classname="poly.test.Account"/>
  <testcase name="CAT05" time="0.885" classname="poly.test.Category"/>
  <testcase name="CAT07" time="0.995" classname="poly.test.Category"/>
  <testcase name="CAT06" time="1.111" classname="poly.test.Category"/>
  <testcase name="CAT08" time="0.901" classname="poly.test.Category"/>
  <testcase name="CAT09" time="0.858" classname="poly.test.Category"/>
  <testcase name="CAT04" time="0.953" classname="poly.test.Category"/>
  <testcase name="LOG05" time="4.563" classname="poly.test.Login"/>
  <testcase name="LOG06" time="1.456" classname="poly.test.Login"/>
  <testcase name="LOG07" time="1.542" classname="poly.test.Login"/>
  <testcase name="LOG08" time="0.286" classname="poly.test.Login"/>
  <testcase name="LOG09" time="1.526" classname="poly.test.Login"/>
  <testcase name="LOG01" time="1.157" classname="poly.test.Login">
    <failure type="java.lang.AssertionError" message="expected [404 HTML Template by Colorlib] but found [HTTP Status 500 – Internal Server Error]">
      <![CDATA[java.lang.AssertionError: expected [404 HTML Template by Colorlib] but found [HTTP Status 500 – Internal Server Error]
at poly.test.Login.LOG01(Login.java:31)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- LOG01 -->
  <testcase name="LOG02" time="0.084" classname="poly.test.Login">
    <failure type="java.lang.AssertionError" message="expected [http://localhost:8080/PS09298ASM/login.htm] but found [http://localhost:8080/PS09298ASM/index.jsp/manage/product.htm]">
      <![CDATA[java.lang.AssertionError: expected [http://localhost:8080/PS09298ASM/login.htm] but found [http://localhost:8080/PS09298ASM/index.jsp/manage/product.htm]
at poly.test.Login.LOG02(Login.java:37)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- LOG02 -->
  <testcase name="PRO06" time="0.854" classname="poly.test.Product"/>
  <testcase name="PRO10" time="1.598" classname="poly.test.Product"/>
  <testcase name="PRO12" time="1.798" classname="poly.test.Product"/>
  <testcase name="PRO14" time="1.684" classname="poly.test.Product">
    <failure type="java.lang.AssertionError" message="expected [tuong1] but found [cam ep11]">
      <![CDATA[java.lang.AssertionError: expected [tuong1] but found [cam ep11]
at poly.test.Product.PRO14(Product.java:94)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- PRO14 -->
  <testcase name="SALE08" time="0.746" classname="poly.test.Product">
    <failure type="java.lang.AssertionError" message="expected [16] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [16] but found [2]
at poly.test.Product.SALE08(Product.java:105)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- SALE08 -->
  <testcase name="PRO05" time="0.801" classname="poly.test.Product"/>
  <testcase name="ACC05" time="0.868" classname="poly.test.Sale"/>
  <testcase name="SALE06" time="1.977" classname="poly.test.Sale"/>
  <testcase name="SALE07" time="1.57" classname="poly.test.Sale"/>
  <testcase name="SALE08" time="0.883" classname="poly.test.Sale"/>
  <testcase name="CAT04" time="0.838" classname="poly.test.Sale"/>
</testsuite> <!-- Regression suite 1 -->
